R/stacking_weights_crps.R

```diff
@@ -32,7 +32,7 @@
 #' 
 #' @param dirichlet_alpha prior for the weights. Default is 1.001
 #' 
-#' @import data.table
+#' @importFrom data.table setDT 
```

Comment:
can you add rbindlist please?
the reason I imported the whole data.table is that I am not sure how all the functions like :=, order etc.. do work. Does `:=` just work? and all others you would reference explicitly by data.table:: ?

Reply:
you need to not import the whole thing. you can import `:=` explicitly. I suggest review carefully.

Reply:
its not used in the function so no.

Reply:
yes you can import `:=` etc explicitly - good idea to check through.

Reply:
you added rbindlist in your code suggestions ;)

Reply:
in the example?

Reply:
oooh. retracted. thanks :)